kbase-extension/static/kbase/js/widgets/narrative_core/upload/importSetup.js

```diff
@@ -6,13 +6,14 @@ define([
     'util/string',
     'common/runtime',
     'StagingServiceClient',
-], (Promise, Jupyter, Config, APIUtil, StringUtil, Runtime, StagingServiceClient) => {
+    './importErrors',
+], (Promise, Jupyter, Config, APIUtil, StringUtil, Runtime, StagingServiceClient, Error) => {
     'use strict';
     const uploaders = Config.get('uploaders');
     const bulkIds = new Set(uploaders.bulk_import_types);
 
     /**
-     *
+     * Fetches
```

Comment:
... fetches what?? Don't leave us hanging!

Reply:
Hopefully it fetches my mind that was left behind after putting the rest of this together.

Reply:
Added info about what's fetched.